packages/atomic/src/components/search/result-template-components/atomic-result-timespan/atomic-result-timespan.tsx

```diff
@@ -0,0 +1,98 @@
+import {Schema, StringValue} from '@coveo/bueno';
+import {Result, ResultTemplatesHelpers} from '@coveo/headless';
+import {Component, Prop, State} from '@stencil/core';
+import dayjs from 'dayjs';
+import {DurationUnitType} from 'dayjs/plugin/duration';
+import {InitializeBindings} from '../../../../utils/initialization-utils';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+import {ResultContext} from '../result-template-decorators';
+import duration from 'dayjs/plugin/duration';
+dayjs.extend(duration);
+
+/**
+ * The `atomic-result-timespan` component renders the duration of a number result field.
```

Comment:
```suggestion
 * The `atomic-result-timespan` component renders a target result number field value as a duration.
```
In the index, does the number have to be the number of milliseconds since unix epoch?